packages/mjml-document/src/index.js

```diff
@@ -0,0 +1,11 @@
+import { BodyComponent } from 'mjml-core'
+
+export default class MjDocument extends BodyComponent {
+  static componentName = 'mjml'
+
+  static endingTag = false
+
+  render() {
+    return ''
+  }
+}
```

Comment:
I'm not really happy with this solution here, body component expect to render something, it's just a hack at the end to make it work :/ 

I guess we could add something like a "mock-class" inside MJML just here to list `mjml` attributes + dependencies but it doesn't really need to have it's own package and should live inside mjml-core